io.github.lethal_guitar.RigelEngine.yaml

```diff
@@ -0,0 +1,36 @@
+app-id: io.github.lethal_guitar.RigelEngine
+
+runtime: org.freedesktop.Platform
+sdk: org.freedesktop.Sdk
+runtime-version: "21.08"
+
+command: run.sh
+
+rename-appdata-file: rigelengine.metainfo.xml
+rename-desktop-file: rigelengine.desktop
+rename-icon: rigelengine
+
+finish-args:
+  - --device=all # to access game controllers
+  - --filesystem=~/rigelengine-game-data:ro
```

Comment:
I suggest using one of the locations your app has write access by default, like $XDG_DATA_HOME.

Reply:
Hi @barthalion, thanks for your review.
I don't need write access, I just need a read-only location that's convenient for the user to place some files into. But they only need to do that once.

I've had a look at other apps similar to mine, i.e. game engines which need user-provided game data, and found the following approaches:

* [io.github.fabiangreffrath.Doom](https://github.com/flathub/io.github.fabiangreffrath.Doom/blob/fa194e02405fb4b1d59b9ceea3d0423b0fa86995/io.github.fabiangreffrath.Doom.json#L14) uses `home:ro`. This is how I originally did it, but changed to the current approach based on https://github.com/flathub/flathub/pull/3170#discussion_r878125921.
* [net.dengine.Doomsday](https://github.com/flathub/net.dengine.Doomsday/blob/0a63af4d33e44caa258d525dc798e994fc5f505e/net.dengine.Doomsday.json#L13) uses `home`, which seems too broad as I don't need write access.
* [org.openttd.OpenTTD](https://github.com/flathub/org.openttd.OpenTTD/blob/68647fe5bd8d8e43cb3e54dac8bea9d73299e6c6/org.openttd.OpenTTD.yaml#L10) also uses `home`.

So it actually seems a common approach for apps similar to mine to map the entire home directory. My current solution is more restricted, and thus should be preferable if I understand correctly. The current solution also seems more user-friendly to me compared to using `XDG_DATA_HOME`, since the latter is inside a hidden folder whereas my approach uses a regular folder. This makes a difference for me because I want my project to be easy to use for less technical users as well. Most graphical file managers do not show dot-files by default, so using `XDG_DATA_HOME` adds an extra step for the user.

Does that make sense?

Reply:
Looking at some more examples, I see that [EDuke32](https://github.com/flathub/com.eduke32.EDuke32/blob/0a257174e16be13913375647e40b24aea5609ee7/com.eduke32.EDuke32.yaml#L13) uses a subdirectory of `XDG_DOCUMENTS`. I would also be fine with that approach, since the documents folder is not a hidden folder.

Reply:
Yeah, I can live with xdg-documents. I'm just not a fan of top-level directories in $HOME.